src/converters/convert.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Given an input is a converter, it will return a converted result.
+ *
+ * @param {object} input - The input to be converted.
+ * @param {boolean} defaultValue - Default value to use if conversion fails.
+ * @param {string} name - The name of of what is converted.
+ *
+ * @returns {object} - The converted result.
+ */
+export default function convert(...converters) {
+    if (!converters.length) {
+        throw new Error('You need to use at least one converter.');
+    }
+
+    return (input, defaultValue, name) => {
+        const result = converters.map((converter) => {
+            return converter(input, defaultValue, name);
```

Comment:
This can be written as 
const result = converters.map((converter) => converter(input, defaultValue, name));